tpgtools/overrides/bigqueryreservation/beta/assignment.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- type: USE_DCL_ID
```

Comment:
What would happen if we changed the default behavior to be this? Do you have a sense of how many overrides we would need to keep current functionality of resources?

Reply:
I'd have to check if the DCL can handle locational resource splits, and then if we have differed from the DCLID in the other already implemented resources. Other than those, the DCLID could be default